meetings/working-groups/roles/roles-2022-11-10.md

```diff
@@ -225,10 +225,10 @@ modopt(Role<T>) UnderlyingType M<T>(modopt(Role<T>) UnderlyingType role)
 One benefit of this approach is that modopt is allowed anywhere a type is allowed:
 `List<modopt(Role<int>) UnderlyingType>`
 
-A downside of this approach is that call sites need to spell out the entire signature, including modopts.
-Also, this implies that changing API from underlying type to role is a breaking change.  
+A downside of this approach is that call sites need to spell out the entire signature, including modopts.  
+Also, this implies that changing API from underlying type to role is a breaking change.   
```

Comment:
Trailing spaces.

```suggestion
A downside of this approach is that call sites need to spell out the entire signature, including modopts.
Also, this implies that changing API from underlying type to role is a breaking change.
```

Reply:
The spaces are intentional, they cause markdown to put linebreaks in.

Reply:
TIL, thanks.